tt_metal/hw/ckernels/grayskull/common/inc/ckernel.h

```diff
@@ -267,36 +332,169 @@ inline void cfg_rmw_gpr(uint32_t cfg_addr32, uint32_t cfg_shamt, uint32_t cfg_ma
     cfg_rmw(cfg_addr32, cfg_shamt, cfg_mask, wrdata);
 }
 
+inline void mailbox_write(const uint8_t thread, const uint32_t data)
```

Comment:
Are these use in the new set of GS LLKs? Or are these for debug infra?

Reply:
These are for debug infra